app/connectors/PenaltiesConnector.scala

```diff
@@ -62,6 +62,30 @@ class PenaltiesConnector @Inject()(httpClient: HttpClient,
     }
   }
 
+  def getMultiplePenaltiesForPrincipleCharge(penaltyId: String, enrolmentKey: String)
+                                            (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[JsValue]] = {
+    val startOfLogMsg: String = "[PenaltiesConnector][getMultiplePenaltiesForPrincipleCharge] -"
+    httpClient.GET[HttpResponse](
+     appConfig.multiplePenaltyDataUrl(penaltyId,enrolmentKey)
+    ).map( response =>
+        response.status match {
+          case OK =>
+            logger.debug(s"$startOfLogMsg OK response returned from Penalties backend for penalty with ID: $penaltyId and enrolment key $enrolmentKey")
+            Some(response.json)
+          case NO_CONTENT =>
+            logger.info(s"$startOfLogMsg Returned 204 from Penalties backend - with body: ${response.body}")
+            None
+          case _ =>
+            logger.warn(s"$startOfLogMsg Returned unknown response ${response.status} with body: ${response.body}")
+            None
```

Comment:
Couldn't we have a parser for this, and this will then prevent you from handling it in the service?